code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -73,10 +73,11 @@
 
 /obj/item/weapon/gun/projectile/automatic/sts35/update_icon(var/ignore_inhands)
 	..()
-	if(istype(ammo_magazine,/obj/item/ammo_magazine/SVD))

```

Comment:
The correct icon for 5.56 mags is the icon that used to be 7.62, because of the arbitrary caliber size of the guns. The magazine sprites were made for those specific guns, and labeled with the old caliber. From my own testing of this PR, the magazines used match both the damage and icon expected for the guns in question.

This's keeps all the sprites and damage in order, as it was before any of the recent gun changes. I checked, manually, that the sprites work.